guides/advanced_guides/known_limitations.md

```diff
@@ -0,0 +1,27 @@
+# Known limitations
+
+MeiliSearch has known limitations, some of which may be addressed in future versions
+
+## Design limitations
+
+### Database size
+
+MeiliSearch uses two database: one for storage and the second one for updates.
+Default database maximum size is 100GiB.
+MeiliSearch uses LMDB under the hood as the storage engine. On launch, LMDB need to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory, which is different from the real memory. You can read more [about storage](/resources/about_storage.md).
```

Comment:
```suggestion
MeiliSearch uses LMDB under the hood as the storage engine. On launch, LMDB needs to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory, which is different from real memory. You can read more [about storage](/resources/about_storage.md).
```

Reply:
👍